### ISSUE TITLE
AWS::ECS::Service DeploymentCircuitBreaker Not Supported By Linter

### ISSUE THREAD

**Issue by MattGirolami** [aws-cloudformation/cfn-python-lint#1979]
*cfn-lint version: (`cfn-lint --version`)*

0.48.2

*Description of issue.*

The DeploymentCircuitBreaker property of ECS Services is flagged as being an invalid property, `[cfn-lint] E3002: Invalid Property Resources/Service/Properties/DeploymentCircuitBreaker` 

Please provide as much information as possible:
* Template linting issues:

```yaml

  Service:
    Type: AWS::ECS::Service
    Properties:
      Cluster: !Ref Cluster
      DeploymentCircuitBreaker:
        Enable: True
        Rollback: False
      DesiredCount: 2
      LaunchType: FARGATE
      PropagateTags: SERVICE
      NetworkConfiguration:
        AwsvpcConfiguration:
          AssignPublicIp: DISABLED
          Subnets:
            - !Ref PrivateSubnetId1
            - !Ref PrivateSubnetId2
      TaskDefinition: !Ref TaskDefinition
```

* Official Documentation

See AWS documentation on this property https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-service-deploymentcircuitbreaker.html

Blog post announcing this feature: https://aws.amazon.com/blogs/containers/announcing-amazon-ecs-deployment-circuit-breaker/

  * Validate if the issue still exists with the latest version of `cfn-lint` and/or the latest Spec files
  
  Exists with v 0.48.2
  
* Feature request:
  * Please provide argumentation about the missing feature. Context is key!
  
 Please add support for this documented ECS Service property. The DeploymentCircuitBreaker provides functionality to detect if an ECS service fails to stabilize (ex: containers crash at start up) and will quickly fail the deployment and can automatically roll back to the previous deployment. Without this property, an ECS service will not be rolled back until the Cloudformation Stack's timeout is reached. 
  
 AWS has recently (as of February 25th) fixed a bug with this feature and Cloudformation, see: https://github.com/aws/containers-roadmap/issues/1205

Cfn-lint uses the [CloudFormation Resource Specifications](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html) as the base to do validation. These files are included as part of the application version. Please update to the latest version of `cfn-lint` or update the spec files manually (`cfn-lint -u`)

This persists after running `cfn-lint -u`

**Comment by PatMyron** [comment]
I think [`DeploymentCircuitBreaker`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-service-deploymentcircuitbreaker.html) falls under [`AWS::ECS::Service.DeploymentConfiguration`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-service-deploymentconfiguration.html)
